session/session.go

```diff
@@ -179,6 +190,22 @@ func (s *Session) Activate(ctx context.Context, i *identity.Identity, c lifespan
 	s.Identity = i
 	s.IdentityID = i.ID
 
+	agent := requestHeaders["User-Agent"]
```

Comment:
It would be better to use `Get` here, no?

Reply:
yes, although `User-Agent` header has multiple values since it appears to be broken into an array of strings delimited by the whitespace char. As a result, `Get` would get only the first bit of the UA